src/main/webapp/help-workspace_de.html

```diff
@@ -1,5 +1,5 @@
 <div>
-  Wenn Quell- und Zielprojekt auf unterschiedlichen Slave-Knoten
+  Wenn Quell- und Zielprojekt auf unterschiedlichen Agent-Knoten
```

Comment:
Could we add a TODO in this file? It also still says `Master` for the built-in node, which does not have a German term yet.

Reply:
Done